libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
All of these are class-level conveniences to make working at the class level cleaner.